### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by vcastellm**
Update the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,26 @@
 
 ![Banner](.github/banner.jpg)
-[![codecov](https://codecov.io/gh/0xPolygon/polygon-edge/branch/develop/graph/badge.svg?token=PXEADRC1IW)](https://codecov.io/gh/0xPolygon/polygon-edge)
+
+## Update: Edge v0.8 is here!  
+
+Developers at Polygon Labs have been hard at work gathering and incorporating community feedback into the Edge client and a new version of Edge, v0.8, is here with several new features!  Check out the Release Notes included with the release to find out more!
```
Comment (Stefan-Ethernal) [inline README.md]: Is this spare space here?

```suggestion
Developers at Polygon Labs have been hard at work gathering and incorporating community feedback into the Edge client and a new version of Edge, v0.8, is here with several new features! Check out the Release Notes included with the release to find out more! 
```

**Review by zuiris — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
 
 ## Documentation 📝
 
 If you'd like to learn more about the Polygon Edge, how it works and how you can use it for your project,
-please check out the **[Polygon Edge Documentation](https://docs.polygon.technology/docs/edge/overview/)**.
+please check out the **[Polygon Supernets Documentation](https://wiki.polygon.technology/docs/supernets/get-started/what-are-supernets)**.
+
+## Disclaimer
+
+As this project evolves, the Polygon Labs developer team will focus on the latest version of the Edge client and does not plan to support Edge 0.6 or lower.  It is highly recommended that you upgrade to the newest version with the most up-to-date features and fixes.  Users that want to stay on 0.6 or below, can continue to do so.  The repo will continue to exist and users can fork it and do with it as they please, subject to applicable open-source license terms.
```
Comment (Stefan-Ethernal) [inline README.md]: Double space?
```suggestion
As this project evolves, the Polygon Labs developer team will focus on the latest version of the Edge client and does not plan to support Edge 0.6 or lower. It is highly recommended that you upgrade to the newest version with the most up-to-date features and fixes. Users that want to stay on 0.6 or below, can continue to do so. The repo will continue to exist and users can fork it and do with it as they please, subject to applicable open-source license terms. 
```

**Review by Stefan-Ethernal — APPROVED** [review]
Cannot remember about those double spaces, is it related to the markdown syntax? :thinking: 

Anyways, LGTM.

**Comment by vcastellm** [comment]
> Cannot remember about those double spaces, is it related to the markdown syntax? 🤔
> 
> Anyways, LGTM.

Nah that's copy paste from Docs 😛 thx!

**Review by goran-ethernal — APPROVED** [review]

**Review by ferranbt — APPROVED** [review]

**Review by Nemanja0x — APPROVED** [review]

**Pull request merged**